core/state_transition.go

```diff
@@ -75,6 +68,41 @@ type Message interface {
 	Data() []byte
 }
 
+// ExecutionResult includes all output after executing given evm
```

Comment:
Wondering if we should move this to `vm`? Seems a bit clearer to type `vm.ExecutionResult` vs. `core.EvexutionResult`. I

Reply:
The reason to put this definition here is: after raw evm execution(we get the `ret` and `vmerr`), we still need to apply some additional rules(e.g. refund gas), so the `usedGas` field has to be updated in the `core` scope. 

Btw now all my VPN nodes are blocked :P. Now I can't login my `discord` account(also the reason for missing standup). Sorry about it.